src/App.tsx

```diff
@@ -1,26 +1,146 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { USER_ID, addTodo, deleteTodo, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { FilterBy } from './utils/FilterBy';
+import { ErrorMessage } from './components/Error';
+import { ErrorTypes } from './types/ErrorTypes';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
```

Comment:
try to rewrite logic without using the additional state for todos

Reply:
feel free to ask for some help in the chat

Reply:
Agreed with Denys. Please fix it.